finagle/h2/src/main/scala/com/twitter/finagle/buoyant/h2/H2ApplicationProtocol.scala

```diff
@@ -0,0 +1,35 @@
+package com.twitter.finagle.buoyant.h2
+
+import com.twitter.finagle.Stack.Module
+import com.twitter.finagle.ssl.ApplicationProtocols
+import com.twitter.finagle.transport.Transport
+import com.twitter.finagle.{ServiceFactory, Stack}
+import io.netty.handler.ssl.ApplicationProtocolNames
+
+object H2ApplicationProtocol {
+
+  val role = Stack.Role("ApplicationProtocol")
```

Comment:
This is in finagle-h2 which doesn't depend on linkerd-core.  I'm open to refactoring this up in the future if it's used elsewhere.